include/core/rheological_model.h

```diff
@@ -102,7 +102,7 @@ class Newtonian : public RheologicalModel<dim>
    * parameter to fit the virtual method in RheologicalModel.
    */
   double
-  get_viscosity(const double &shear_rate_magnitude) override
+  get_viscosity(const double & /*shear_rate_magnitude*/) override
```

Comment:
Right we can do this! We could also just take the comment off, and specify it in the method header